client/blocks/reader-avatar/index.jsx

```diff
@@ -32,7 +43,7 @@ const ReaderAvatar = ( { author, siteIcon, feedIcon, siteUrl, siteIconSize = 96,
 		};
 	}
 
-	let hasSiteIcon = !! ( fakeSite && fakeSite.icon );
+	let hasSiteIcon = !! ( fakeSite && fakeSite.icon.img );
```

Comment:
was this a bug from before? 

Also should it be `!! ( fakeSite ^^ fakeSite.icon && fakeSite.icon.img ) `?